src/gc.h

```diff
@@ -11,20 +11,24 @@
 
 #define V7_DISABLE_GC
 
-#define MARK(p) (* (uintptr_t *) &(p) |= 1)
+#define MARK(p) (* (unsigned long *) &(p) |= 1)
```

Comment:
Got it. Restored the old logic with uintptr_t, and typedefed uintptr_t to unsigned long on Win32.
Keep GC disabled coz unit test still coredumps.